requirements/base.txt

```diff
@@ -4,4 +4,12 @@
 #
 #    pip-compile --output-file requirements/base.txt requirements/base.in
 #
-Django==1.10.5
+amqp==2.1.4               # via kombu
+billiard==3.5.0.2         # via celery
+celery==4.0.2
+django-model-utils==2.6.1
```

Comment:
The .txt requirement files are autogenerated (by pip-tools) from the .in files for local development to enable replicable builds.  When edx-platform installs this app, it will use setup.py (which has loosely versioned or unversioned dependencies).

tox is set up to multiple runs against multiple versions of key requirements, particularly python, django, and celery (which affects several of celery's dependencies as well).

django-model-utils is not currently tested with different versions.  The library is currently unpinned in base.in (because it doesn't require a particular version), and shouldn't need to be upgraded separately when edx-platform wants to update its requirement.